src/openrct2/ride/Vehicle.cpp

```diff
@@ -7631,7 +7650,7 @@ void Vehicle::UpdateReverserCarBogies()
  */
 bool Vehicle::UpdateMotionCollisionDetection(const CoordsXYZ& loc, uint16_t* otherVehicleIndex)
 {
-    if (HasUpdateFlag(VEHICLE_UPDATE_FLAG_COLLISION_DISABLED))
+    if (HasUpdateFlag(VEHICLE_UPDATE_FLAG_COLLISION_DISABLED) || HasUpdateFlag(VEHICLE_UPDATE_FLAG_SINGLE_CAR_POSITION))
```

Comment:
Any reason you are doing this rather than just setting `VEHICLE_UPDATE_FLAG_COLLISION_DISABLED` as well as `VEHICLE_UPDATE_FLAG_SINGLE_CAR_POSITION`

Reply:
I can do that yes, that sounds like a better idea. Will it not interfere with any other code if I clear the `VEHICLE_UPDATE_FLAG_COLLISION_DISABLED`-flag again after `UpdateTrackMotion`? I am not sure when and where this flag is used elsewhere.

Reply:
I know it's used for vehicle placement (which is not an issue). Have a look see if it's set for anything else

Reply:
The only case I can find is the Giga Coaster cablelift, where it is enabled permanently. I don't think `UpdateMotionCollisionDetection` is called for the cablelift though, so it seems safe to clear it. :)